### DISCUSSION TITLE
Page Not fully Rendered in browser

### DISCUSSION THREAD

**Discussion by Previesam** [harttle/liquidjs#293] [General]
Well done All,

I have this issue, installed liquid js, added html to my index.liquid file and did all the necessary express setup, the page renders and everything is supposed to be fine except that the page loads but only the part of the page that is visible displays when I scroll down on the page I find spaces for content that have been loaded but are not displayed.

I have tried loading the html in static mode without liquid and it works fine the element animates in as I scroll down on the pages, which is not the case when I was using the template.

I also noticed that in the template mode, when I start resizing the page manually, the elements that were not showing before will start showing.

Please what could be the issue

**Comment by harttle** [comment]
Can you post a minimal code snippet to reproduce the problem? "the page loads but only the part of the page that is visible" is technically not a LiquidJS problem, it's will be useful to identify the problem if you post the difference of the generated HTML code before and after use LiquidJS.
